installer/install.sh

```diff
@@ -13,11 +16,16 @@ readonly VER_LT=2
 oc_install_dir="/usr/local/bin"
 oc_version_comparison=${VER_LT}
 
-#Minimal version requirements
+#####################################################################
+# Minimal version requirements                                      #
+#####################################################################
 readonly MIN_PYTHON_VERSION=2.7
 readonly MIN_ANSIBLE_VERSION=2.6
 readonly MIN_OCP_CLIENT_TOOL=3.9
 
```

Comment:
For example, it probably should be in a properties file. However, now it is already centralized and if we need change a version we can find the place easier than before. Once, just one place needs to be changed.